### PR TITLE
Added landing page for fall hacks 2020

### PR DESCRIPTION
**Pull request by SpartanApple**

### CODE REVIEW

File: csss-site/src/events/templates/events/fall_hacks2020.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{% extends "csss/new_header.html" %}
+
+{% block content %}
+{{ block.super }}
+    <title>Fall Hacks 2020</title>
+
+<style>
```
Comment (modernNeo) [inline csss-site/src/events/templates/events/fall_hacks2020.html:7]: please put these in a separate css file. I believe that is best practice.

File: csss-site/src/events/templates/events/fall_hacks2020.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  border: 1px solid black;
+  border-collapse: collapse;
+}
+</style>
+
+
+ 
+
+      <header>
+	      <h1> <a href="{{URL_ROOT}}events/fall_hacks_submissions2020">Project Page of submissions from the Computing Science Student Society Fall Hacks 2020 submissions<a></h1>
```
Comment (modernNeo) [inline csss-site/src/events/templates/events/fall_hacks2020.html:18]: bit redundant much, no?
why not `Computing Science Student Society Fall Hacks 2020 submissions`?

also, why does a link need a header tag?

File: csss-site/src/events/templates/events/fall_hacks_submissions2020.html [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+
+  </head>
+  <body>
+
+ 
+
+    <section id='above-fold'>
+
+      <header>
+        <h1>Project Page of submissions from the Computing Science Student Society Fall Hacks 2020 submissions</h1>
```
Comment (modernNeo) [inline csss-site/src/events/templates/events/fall_hacks_submissions2020.html:22]: needs to be `<h1 class="title">` if you actually want the text to be bolder and bigger

File: csss-site/src/csss/templates/csss/new_header.html [hunk 4]
```diff
@@ -97,6 +97,8 @@
             <a href="{{URL_ROOT}}events/board_games" class="navbar-item">What are Board Games Night?</a>            
             <a href='{{ URL_ROOT}}events/frosh/' class="navbar-item">CSSS FroshWeek</a>
             <a href="{{URL_ROOT}}events/mountain_madness2020" class="navbar-item">Mountain Madness 2020</a>          
+            <a href="{{URL_ROOT}}events/mountain_madness2021" class="navbar-item">Mountain Madness 2021</a>          
+            <a href="{{URL_ROOT}}events/fall_hacks2020" class="navbar-item">Fall Hacks 2020 Projects</a>
```
Comment (modernNeo) [inline csss-site/src/csss/templates/csss/new_header.html:101]: can you order these chronologically?

**Review by modernNeo — CHANGES_REQUESTED** [review]

File: csss-site/src/csss/templates/csss/new_header.html (hunk 4 above)
Comment (SpartanApple) [reply csss-site/src/csss/templates/csss/new_header.html:101]: No, the frosh events are not ordered chronologically, they should be grouped by event type/name then by date.

File: csss-site/src/events/templates/events/fall_hacks2020.html (hunk 1 above)
Comment (modernNeo) [reply csss-site/src/events/templates/events/fall_hacks2020.html:6]: you hit resolve but did not change it?

**Review by modernNeo — CHANGES_REQUESTED** [review]

File: csss-site/src/events/templates/events/fall_hacks2020.html (hunk 1 above)
Comment (SpartanApple) [reply csss-site/src/events/templates/events/fall_hacks2020.html:6]: Correct, I think it's more convent in this file. I did remove the title.

File: csss-site/src/events/templates/events/fall_hacks2020.html [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+{{ block.super }}
+
+<style>
+table, th, td {
+  border: 1px solid black;
+  border-collapse: collapse;
+}
+</style>
+	 <a href="{{URL_ROOT}}events/fall_hacks_submissions2020">Computing Science Student Society Fall Hacks 2020 submissions<a>
+  </body>
```
Comment (modernNeo) [inline csss-site/src/events/templates/events/fall_hacks2020.html:13]: where is the opening body tag?

**Review by modernNeo — CHANGES_REQUESTED** [review]

File: csss-site/src/events/templates/events/fall_hacks2020.html [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+{% block content %}
+{{ block.super }}
+
+<style>
+table, th, td {
+  border: 1px solid black;
+  border-collapse: collapse;
+}
+</style>
+	 <a href="{{URL_ROOT}}events/fall_hacks_submissions2020">Computing Science Student Society Fall Hacks 2020 submissions<a>
```
Comment (modernNeo) [inline csss-site/src/events/templates/events/fall_hacks2020.html:12]: Please add a header tag for the string `Computing Science Student Society Fall Hack 2020`

and then below it just update the link to `submissions`

File: csss-site/src/csss/templates/csss/new_header.html (hunk 4 above)
Comment (modernNeo) [inline csss-site/src/csss/templates/csss/new_header.html:101]: change `Fall Hacks 2020 Projects` to `Fall Hacks 2020`

File: csss-site/src/events/templates/events/fall_hacks_submissions2020.html [hunk 7]
```diff
@@ -0,0 +1,329 @@
...
+  border: 1px solid black;
+  border-collapse: collapse;
+}
+</style>
+
+  </head>
+  <body>
+    <section id='above-fold'>
+      <header>
+        <h1 class="title">Project Page of submissions from the Computing Science Student Society Fall Hacks 2020</h1>
```
Comment (modernNeo) [inline csss-site/src/events/templates/events/fall_hacks_submissions2020.html:18]: maybe update that string to `Computing Science Student Society Fall Hacks 2020 Submissions`?

File: csss-site/src/events/templates/events/fall_hacks_submissions2020.html [hunk 8]
```diff
@@ -0,0 +1,329 @@
+{% extends "csss/new_header.html" %}
+
+{% block content %}
+{{ block.super }}
+    <title>Fall Hacks 2020</title>
```
Comment (modernNeo) [inline csss-site/src/events/templates/events/fall_hacks_submissions2020.html:5]: you sure this is correct? when I load the page at https://dev.sfucsss.org/PR-280/events/fall_hacks_submissions2020 , I see "SFU CSSS" on the tab.

**Review by modernNeo — CHANGES_REQUESTED** [review]

File: csss-site/src/events/templates/events/fall_hacks_submissions2020.html (hunk 8 above)
Comment (modernNeo) [reply csss-site/src/events/templates/events/fall_hacks_submissions2020.html:5]: seems the issue is that there is a title in the template, and first title is the one that makes a difference. will remove it.

File: csss-site/src/events/templates/events/fall_hacks2020.html [hunk 9]
```diff
@@ -2,12 +2,15 @@
...
-
 <style>
 table, th, td {
   border: 1px solid black;
   border-collapse: collapse;
 }
 </style>
-	 <a href="{{URL_ROOT}}events/fall_hacks_submissions2020">Computing Science Student Society Fall Hacks 2020 submissions<a>
+
+<html>
```
Comment (modernNeo) [inline csss-site/src/events/templates/events/fall_hacks2020.html:12]: why are you adding an html tag in a template? 😂 its already in here.

**Review by modernNeo — CHANGES_REQUESTED** [review]

File: csss-site/src/events/templates/events/fall_hacks2020.html (hunk 6 above)
Comment (modernNeo) [reply csss-site/src/events/templates/events/fall_hacks2020.html]: @SpartanApple, once this is fixed, I think I will merge

**Pull request merged**